joblib/test/test_numpy_pickle.py

```diff
@@ -358,6 +502,10 @@ def test_joblib_pickle_across_python_versions():
                      # compatibility alias for .tobytes which was
                      # added in 1.9.0
                      np.arange(256, dtype=np.uint8).tostring(),
+                     # np.matrix is a subclass of nd.array, here we want
```

Comment:
There are multiple instances of this (probably was like this originally ...) do a search and replace.